### PR TITLE
Current limit msg

### PR DESCRIPTION
**Pull request by awesomebytes**
We need this message to be able to send events from the web/tablet (that just publishes to topics) so the package: https://github.com/pal-robotics/learn_from_robot
Can read from it and set currents into the motors.

I need to use this today

### CODE REVIEW

**Comment by bmagyar** [comment]
Me gusta :+1:

**Comment by v-lopez** [comment]
I need this asap for building the GUI.

File: pal_common_msgs/msg/JointCurrent.msg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Joints or group name of joints to activate or deactivate
+string joints
```
Comment (adolfo-rt) [inline pal_common_msgs/msg/JointCurrent.msg:2]: actuators, not joints.

How is the notion of an actuator group implemented?, is there a reference implementation?.

**Comment by adolfo-rt** [comment]
I see that there is a great sense of urgency revolving around setting the actuator current limits.

Right now there are three separate mechanisms under development to do this ([context](https://github.com/pal-robotics/pal_msgs/pull/9#issuecomment-47328176)), each different in its own way. I suggest we take a moment to at least discuss the message API, before starting to write code like crazy and then have to perform painful migrations.

We can do this now if you want.

**Comment by awesomebytes** [comment]
I'm merging this. I need it now for using it tomorrow.
I'll take care of all the refactor of it (it's only used by my package and Victors package that I need exactly  NOW). No one will use it as adolfo has another messages that are more correct it seems. But my code is implemented already using this and I have no time to change it now. I'll have plenty of time after the robocup

**Pull request merged**